ckks/bootstrap_params.go

```diff
@@ -33,229 +17,1085 @@ const (
 	Cos2 = SinType(2) // Standard Chebyshev approximation of pow((1/2pi), 1/2^r) * cos(2pi(x-0.25)/2^r)
 )
 
-// Copy return a new BootstrapParams which is a copy of the target
+// BootstrappingParameters is a struct for the default bootstrapping parameters
+type BootstrappingParameters struct {
```

Comment:
I agree. We could do this change in the PR for CtS and StC in the evaluator, as it would also have us redesign the bootstrapping parameters in a way too.